### PR TITLE
Move fonts out of core repo

### PR DESCRIPTION
**Pull request by mmcc**
There's now a font build pipeline: [videojs-font](http://github.com/videojs/font)

### CODE REVIEW

**Comment by gkatsev** [comment]
You mean https://github.com/videojs/font?

**Comment by gkatsev** [comment]
Ugh, the diff for the grunt.js file is horrible.

**Comment by mmcc** [comment]
Ah yeah, forgot to revert the formatting change. I have my editor set to
auto format on save and meant to disable it before messing with core.

File: src/css/_variables.scss [hunk 1]
```diff
@@ -4,7 +4,6 @@ $secondary-text: #F4A460; // currently just used for visited links in errors and
 $primary-bg: #000;
 $secondary-bg: lighten($primary-bg, 35%);
 
-$icon-font-family: 'VideoJS';
```
Comment (heff) [inline src/css/_variables.scss]: Should we not still set this here anyway? How might someone override it with their own? If that's a real use case.

Comment (mmcc) [inline src/css/_variables.scss]: If someone was overriding it with there own, they'd probably want to override all of the icons partial basically (at least that was my thought).

**Comment by heff** [comment]
Should be good to go after the grunt file cleanup.

File: src/css/_variables.scss (hunk 1 above)
Comment (heff) [inline src/css/_variables.scss]: yeah, that makes sense

**Comment by heff** [comment]
Looks good. Pulling in.

**Pull request closed without merging**